src/pages/layer-2.js

```diff
@@ -679,6 +679,15 @@ const Layer2Page = ({ data }) => {
         <p>
           <Translation id="layer-2-use-layer-2-1" />
         </p>
+        <p>
+          NOTE: When bridging over and using layer 2, it is important to note
+          that you will control the address for you EOA account just like on
```

Comment:
I would expand on "EOA" account, e.g. an account where only a single private key controls the funds / users a recovery phrase / etc. given that this warning is probably more for novice than advanced users.